src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMappingProvider.cs

```diff
@@ -16,6 +16,22 @@ public PackageSourceMappingProvider(ISettings settings)
             _settings = settings ?? throw new ArgumentNullException(nameof(settings));
         }
 
+        /// <summary>
+        /// Supports the disabling of saving to disk for any <see cref="ISettings"/> changes.
+        /// </summary>
+        /// <param name="shouldSkipSave">True to avoid saving any changes to disk and only modify the <see cref="ISettings"/> in memory.
```

Comment:
I removed it since our policy is "if you don't write content for a param, then don't have a param comment at all". Didn't have anything interesting to write for it.